netbox/dcim/svg/cables.py

```diff
@@ -366,6 +370,11 @@ def render(self):
 
                 obj_list = {end.parent_object for end in far_ends}
                 parent_object_nodes, far_terminations = self.draw_far_objects(obj_list, far_ends)
+
+                # If there are no far terminations, this segment can't be rendered, so skip.
+                if not far_terminations:
+                    continue
```

Comment:
Can this be obviated by checking for `far_ends` in addition to `links` above?

```
if links and far_ends:
```

Reply:
I think you're right, that does work fine and is a lot more surgical.